### PR TITLE
Version 2.0.0

### PR DESCRIPTION
**Pull request by mathiascode**
Letting this PR sit for one week. If no urgent bugs are reported in that time period, we can merge this, unless there are objections.

@emilazy @melmorabity

### CODE REVIEW

**Comment by mathiascode** [comment]
This PR may end being slightly delayed, depending on which day I come back next week. In any case, I'll prepare a Windows installer when it happens.

**Comment by mathiascode** [comment]
I think we're ready. Here's the final 2.0.0 Windows installer: https://github.com/mathiascode/nicotine-plus/raw/exe/files/windows/Nicotine%2B-2.0.0.exe

**Comment by mathiascode** [comment]
@kiplingw Does this look good to you? I don't have any more additions for the 2.0.0 release personally.

**Comment by kiplingw** [comment]
Well folks, what do you think? Are we ready for a 2.0.0 release?

**Comment by mathiascode** [comment]
I think I'll postpone this slightly until I have created a fix for an issue with distributed searches.

**Comment by mathiascode** [comment]
Distributed searches now work well enough, but I want to get https://github.com/Nicotine-Plus/nicotine-plus/pull/321 included in the 2.0.0 release.

**Comment by mathiascode** [comment]
My current plan (if nothing else comes in the way): finish https://github.com/Nicotine-Plus/nicotine-plus/pull/337 -> start working on and finish https://github.com/Nicotine-Plus/nicotine-plus/issues/335 -> 2.0.0 release

**Comment by mathiascode** [comment]
In the end, the 2.0.0 update turned out a lot larger than anticipated, but we should be much closer to a rock solid Nicotine+ now. :)

Things are looking pretty good for a new release right now, but I suggest anyone interested to still do some final testing of the master branch. I'll start a new unstable PPA build now.

**Comment by kiplingw** [comment]
We could do a 2.0.0 release. But let's wait another two weeks to see if there's any more bugs that are identified in the interim. If we don't find any then what's in the repository by then we can release.

**Comment by mathiascode** [comment]
Sounds good to me!

**Comment by hboetes** [comment]
Perhaps release a beta or an RC? To invite a bigger testing crows?

**Comment by mathiascode** [comment]
I'm not convinced we're going to attract new testers by doing so. The userbase is already small, and most Nicotine+ users probably don't check the website regularly. I'm more inclined to release 2.0.0 soon, and potentially release a 2.0.1 if necessary, to prevent the release cycle from dragging on forever.

**Comment by hboetes** [comment]
True, I've happened to work on another project where the lead dev was a complete perfectionist which basically killed the whole project.

**Comment by kiplingw** [comment]
I think we are ready for a 2.0.0 release. Unless there are any objections, I will merge this today. @mathiascode, for your next PR, bump the version to 2.0.1 so the unstable package is distinguished in people's package managers who have both PPAs installed.

File: NEWS [hunk 1]
```diff
@@ -4,7 +4,7 @@ News
     As per GCS § 6.7, this file contains a list of user-visible noteworthy
     changes. Note that this is not the same as a changelog.
 
-Version 2.0.0 (unreleased)
+Version 2.0.0 (24 June 2020)
```
Comment (kiplingw) [inline NEWS:7]: Bump this to current date.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # Nicotine+
 
-Nicotine+ is a graphical client for the [Soulseek](https://www.slsknet.org/) peer-to-peer file sharing network. It is an attempt to keep [Nicotine](https://web.archive.org/web/20150720173459/http://nicotine.thegraveyard.org/) working with the latest libraries, kill bugs, keep current with the Soulseek protocol, and add some new features that users want and/or need.
+Nicotine+ is a graphical client for the [Soulseek](https://www.slsknet.org/news/) peer-to-peer file sharing network. It is an attempt to keep [Nicotine](https://web.archive.org/web/20150720173459/http://nicotine.thegraveyard.org/) working with the latest libraries, kill bugs, keep current with the Soulseek protocol, and add some new features that users want and/or need.
 
 [Screenshots](files/screenshots/SCREENSHOTS.md)
 
 # Download Nicotine+
-The current stable version of Nicotine+ is 1.4.1. An unstable version 1.4.3 is currently under development and available for testing.
+The current stable version of Nicotine+ is 2.0.0, released on 24 June 2020. See the [changelog](NEWS).
```
Comment (kiplingw) [inline README.md:8]: Bump to current date.

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,9 @@
+nicotine (2.0.0-1) focal; urgency=medium
+
+  * Latest upstream.
+
+ -- Mathias <mail@mathias.is>  Mon, 24 June 2020 13:14:00 +0300
```
Comment (kiplingw) [inline debian/changelog:5]: Bump to current date.

File: files/appdata/org.nicotine_plus.Nicotine.appdata.xml [hunk 4]
```diff
@@ -45,4 +45,7 @@
     <content_attribute id="social-chat">intense</content_attribute>
     <content_attribute id="social-info">mild</content_attribute>
   </content_rating>
+  <releases>
+    <release version="2.0.0" date="2020-06-24" />
```
Comment (kiplingw) [inline files/appdata/org.nicotine_plus.Nicotine.appdata.xml:49]: Bump to current date.

**Comment by mathiascode** [comment]
Done! Let's get https://github.com/Nicotine-Plus/nicotine-plus/pull/390 merged, merge this, and I'll create a Windows installer to include in the GitHub release.

**Comment by hboetes** [comment]
yay! Party! :-D

**Comment by kiplingw** [comment]
2.0.0 being built in stable PPA now.

**Comment by mathiascode** [comment]
Windows installer: https://github.com/mathiascode/nicotine-plus/blob/8d9cd9a039273704f1107e3bde76faa4cac0f793/files/windows/Nicotine+-2.0.0.exe?raw=true

**Comment by mathiascode** [comment]
@kiplingw Sorry for pinging again, but can you attach the Windows installer to the 2.0.0 GitHub release, so we avoid serving a 404 download link on the homepage for too long?

**Comment by kiplingw** [comment]
Done.

**Pull request merged**